conans/test/functional/command/devflow_test.py

```diff
@@ -62,7 +62,7 @@ def test_parallel_folders(self):
         client.run("export-pkg . Pkg/0.1@lasote/testing -bf=../build")
 
         ref = ConanFileReference.loads("Pkg/0.1@lasote/testing")
-        cache_package_folder = client.cache.package_layout(ref).packages()[0]
+        cache_package_folder = client.get_latest_pkg_layout(ref).package()
```

Comment:
This is weird, how can you get the pkg_layout if you don't provide the ``package_id``. It is a bit risky and confusing, maybe the argument should always be a ``pref``, not a ``ref``

Reply:
Yes, that was I was thinking... Ok, let me check all the cases because it's not the only one in the tests

Reply:
The thing I see here is that we force to give a package id for this, then we will need to add more code just to get the list of package_ids and after that the list of package revisions. Old tests don't fit so well with the new cache, they try to get packages folders from just the `ConanFileReference` in many places...